### ISSUE TITLE
Commas in Blog post title cause roleName error

### ISSUE THREAD

**Issue by gerry3246** [Shazwazza/Articulate#254]
Hi Shannon,

We're using Articulate 2.0.1 on a 7.3.1 install. Our content people made a couple of blog posts recently where the title contained a reference to money (i.e. "By doing x, residents in y county saved $20,000"). ON saving the post, the roleName error comes up:
Server error: Contact administrator, see log for full details.
The parameter 'roleName' must not contain commas. Parameter name: roleName

I have searched on this error, most answers indicating that it arises from having a MemberGroup with a comma in th ename. We have none, but it looks like when saving the blog post node, Umbraco is trying to make a new membergroup associated with this node? If so, is there any way to stop that behavior? We don't want membergroups for Blog nodes, and not being able to include commas in a blog post title is not going to be acceptable to our content folks.

Here is the odd part - the post saved and published. No worries, we thought we could just ignore the error, since we dont want membergroups made form blog posts anyway. 
Except, our content person later needed to edit the blog post title. She changed a word, hit save & publish, got the roleName error. In the backoffice, the post title was updated to the new value. Unfortunately, on the public side, the post still has the old (incorrect) value.

I tried incrementing the ClientDependency as well, but that didnt help.

Any idea what I should do now?

**Comment by Shazwazza** [comment]
This is all very odd. Articulate doesn't do anything with members so something else must be going on.

You are also running an old version and should use the latest Articulate version.

Try clearing your cookies. I'm really not sure, maybe you have some custom event handlers trying to do something? or another package. 

You should turn debug="true" in your web.config compilation section so you can see what the full error message is and also check your logs.

**Comment by gerry3246** [comment]
Hi Shannon,

Excellent advice. I do have some event handlers, but not for these document
types.

We did find a work-around. Clicking Save & Publish on a blog post that is
already published and has a comma in the title produces the error. However,
if we first Unpublish the post, then Save & Publish, it works fine without
error.  I reindexed the site, and cleared cache before we found the
workaround; neither seemed to have a positive effect.

The site is legacy; it has been upgraded and modified since v4.10 and it
has become unstable after so many changes. We're building a new site based
on a fresh 7.6 and new database this summer. Hopefully this sort of thing
will become an unpleasant memory.

Given what you said, I dont think Articulate is the cause either. Feel free
to close this issue.

Thanks!
Gerry

On Wed, May 31, 2017 at 12:40 PM, Shannon Deminick <notifications@github.com
